modular_skyrat/modules/chaplain/code/chaplainmodule.dm

```diff
@@ -0,0 +1,23 @@
+// FILE FOR CHAPLAIN RELATED SKYRAT CODE
+
+// Nun outfit
+/obj/item/clothing/head/nun
+	name = "nun headdress"
+	desc = "A cloth headpiece that covers the hair and ears. The iconic uniform of nuns across the galaxy."
+	icon = 'modular_skyrat/modules/chaplain/icons/nun.dmi'
+	worn_icon = 'modular_skyrat/modules/chaplain/icons/nunmob.dmi'
```

Comment:
Clothing icons should be going into modular_skyrat/master_files/icons/(obj or mob)/clothing, please!